data/net/minecraft/world/entity/ai/sensing/HurtBySensor.mapping

```diff
@@ -2,3 +2,5 @@ CLASS net/minecraft/world/entity/ai/sensing/HurtBySensor
 	METHOD doTick (Lnet/minecraft/server/level/ServerLevel;Lnet/minecraft/world/entity/LivingEntity;)V
 		ARG 1 level
 		ARG 2 entity
+	METHOD lambda$doTick$0 (Lnet/minecraft/server/level/ServerLevel;Lnet/minecraft/world/entity/ai/Brain;Lnet/minecraft/world/entity/LivingEntity;)V
+		ARG 2 entity
```

Comment:
``entity`` -> ``attacker``

Reply:
Still relevant, this collides with the outer method parameter.